quiz.js

```diff
@@ -18,6 +18,22 @@ db.dropDatabase();
 // your code begins here
 
 
+// for terminal "use weseros"
+
+db.westeros.insert({
+    name : 'Arryn',
+    motto : 'As High as Honor',
+});
+
+db.westeros.insert({
+    name : 'Stark',
+    motto : 'Winter is Coming',
+});
+
+db.westeros.insert({
+    name : 'Targaryen',
+    motto : 'Fire and Blood',
+});
 
```

Comment:
Mostly right, but the instructions say to call your collection 'houses', not 'westeros'. Plus, since our database is also called 'westeros', we would probably also want to try to avoid confusion by giving our collection a different name.